### DISCUSSION TITLE
【后续】成功逃脱level 0！！！

### DISCUSSION THREAD

**Discussion by M1aoW** [share121/inter-knot#3109] [其他]
![tqe0cq0r](https://github.com/user-attachments/assets/3fd4f2db-b466-4e90-baec-e856abf8a0ad)
### 在我绝望之际居然看黄色的墙皮上看见了箭头，拼了命的跟着箭头的指引突然落入了这个诡异的地方。
这次我很幸运，没多久就有人找到了我，他们穿着防护服，自称是M.E.G.，为我介绍了现在的处境，这里确实是名为backrooms（后室）的地方，和空洞不同的是这地方的特点是有不同的层级，刚刚那黄色墙皮组成的迷宫是level 0，而这里是level 1，进入这里之后是无法出去的，只能在这里生存下去。
他们说来到这里说明我最基本的生存有了保障，他们给了我一瓶灰色杏仁水，说这就是在这个世界生存的保障，各个层级都有可能刷新。这玩意简直不要太好喝，喝完一瓶感觉之前积累的疲惫感瞬间烟消云散，他们表示这种资源不是白来的，并希望我直接加入M.E.G.，在必要的时候做出贡献，就可以为我提供足以生存的物资，在这种不明所以的世界，除了加入他们我别无选择。
之后他们带我进入了Alpha基地，这里目前有留存大量的流浪者，他们带我进入一片营地，为我选择了一个床位，每天定期会给予共三瓶杏仁水，与之相对的是帮助基地搬取物品，清洁卫生。如果有其他需求，可以加入部门或者兵团做出贡献或者进行其他楼层的探索，换取功勋，拥有功勋M.G.E.就会尽可能满足你一切需求，不过每个层级有不同的危险，希望我谨慎考虑。
他们离开的很快，不久就看到另一批人带着刚步入后室的流浪者进入营地，我不禁感慨这真是一个强大的组织，对比新艾利都的防卫军，两者哪边更强大呢。
这里居然能给我的手机充电，今天的运气实在是太好了，基地内禁止拍照，很遗憾我不能给各位看看里面的景观，可以的话我想在这里躺平就好，level 0的那种情况太让人绝望了，居然还是个教学关卡，实在不敢想其他层级到底是个什么样子。
就这样，之前的委托取消了，也警醒各位探索空洞的任何人员，一定要小心无法记录的裂隙，尽可能的不要进入这里。

**Comment by mahjong-tile** [comment]
woc你还活着！<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/7cb87b286c3d062300d31224508c0c9e_8768116406867668880.png" alt="丹恒 惊" width="75" >

**Comment by warning-Kirov-reporting** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by OttoApocalypse01** [comment]
再探再报<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Comment by trifortune** [comment]
！居然不是空洞，打赌输了2000丁尼
